generators/client/templates/react/src/main/webapp/app/config/websocket-middleware.ts.ejs

```diff
@@ -0,0 +1,108 @@
+import * as SockJS from 'sockjs-client';
+import * as Stomp from 'webstomp-client';
+import { Observable, Observer } from 'rxjs';
```

Comment:
we dont have RxJS in dependency defined, where do you get it from?

Reply:
It was coming from another dependency I guess.  `yarn why rxjs` says it comes from `lint-staged#listr`.  

I added it to `package.json` conditionally on websockets.

Reply:
May be we should also ensure we only import what we need from rxjs, live we do for Angular, since rxjs is quite heavy

Reply:
861kb -> 35kb raw size change according to the Webpack visualizer